lib/3scale/api/client.rb

```diff
@@ -103,6 +103,15 @@ def create_service(attributes)
 
       # @api public
       # @return [Hash]
+      # @param [Hash] attributes Service Attributes
+      # @option attributes [String] :name Service Name
+      def update_service(attributes, service_id)
+        response = http_client.put("/admin/api/services/#{service_id}", body: { service: attributes })
```

Comment:
I think for update would be great to use the new `PATCH` method.
And if it is is missing in the http client, add it there.